gdal/src/main/scala/geotrellis/raster/gdal/GDALRasterSource.scala

```diff
@@ -97,7 +97,7 @@ class GDALRasterSource(
     * These resolutions could represent actual overview as seen in source file
     * or overviews of VRT that was created as result of resample operations.
     */
-  lazy val resolutions: List[CellSize] = dataset.resolutions(datasetType).map(_.cellSize)
+  lazy val resolutions: List[CellSize] = dataset.resolutions(GDALDataset.SOURCE).map(_.cellSize)
```

Comment:
Moved this PR into `in-progress` state, since I need to fix comments and probably to add the old version / function that will allow to get resolutions by the `GDALDataset.TYPE`